src/sonic-yang-models/tests/yang_model_tests/tests_config/ntp.json

```diff
@@ -237,5 +237,144 @@
                 ]
             }
         }
+    },
+    "NTP_SERVER_WITH_DEFAULT_MINPOLL_MAXPOLL": {
+        "sonic-ntp:sonic-ntp": {
+            "sonic-ntp:NTP_SERVER": {
+                "NTP_SERVER_LIST": [
+                    {
+                        "server_address": "10.11.12.13",
+                        "minpoll": "6",
+                        "maxpoll": "10"
+                    }
+                ]
+            }
+        }
+    },
+    "NTP_SERVER_WITH_MIN_MINPOLL_MAXPOLL": {
+        "sonic-ntp:sonic-ntp": {
+            "sonic-ntp:NTP_SERVER": {
+                "NTP_SERVER_LIST": [
+                    {
+                        "server_address": "10.11.12.13",
+                        "minpoll": "3",
+                        "maxpoll": "4"
+                    }
+                ]
+            }
+        }
+    },
+    "NTP_SERVER_WITH_MINPOLL_MAX_MAXPOLL": {
+        "sonic-ntp:sonic-ntp": {
+            "sonic-ntp:NTP_SERVER": {
+                "NTP_SERVER_LIST": [
+                    {
+                        "server_address": "10.11.12.13",
+                        "minpoll": "12",
+                        "maxpoll": "17"
+                    }
+                ]
+            }
+        }
+    },
+    "NTP_SERVER_NEG_WITHOUT_MINPOLL": {
+        "sonic-ntp:sonic-ntp": {
+            "sonic-ntp:NTP_SERVER": {
+                "NTP_SERVER_LIST": [
+                    {
+                        "server_address": "10.11.12.13",
+                        "maxpoll": "11"
+                    }
+                ]
+            }
+        }
+    },
+    "NTP_SERVER_NEG_WITHOUT_MINPOLL_INVALID_MAXPOLL": {
+        "sonic-ntp:sonic-ntp": {
+            "sonic-ntp:NTP_SERVER": {
+                "NTP_SERVER_LIST": [
+                    {
+                        "server_address": "10.11.12.13",
+                        "maxpoll": "5"
```

Comment:
What's the difference between this test case and the one above?

Reply:
Ah, never mind, I see that since the default minpoll is 6, this returns an error.